### ISSUE TITLE
Footer link accessability

### ISSUE THREAD

**Issue by ChrRaz** [eriicali/aboutme#8]
The social media links in the footer currently use javascript to redirect the browser.
This means that the links cannot be found by screen readers and you can't open the links in a new tab.
Instead the svg's should be inside `a` tags with an appropriate `href`.
